Automattic/wp-calypso
# Activity Logo: Use custom design for 'Server' actor

See https://github.com/Automattic/wp-calypso/issues/71165
See conversation in pdWQjU-b6-p2

## Proposed Changes

Adds a custom design for the 'Server' actor:

<img width="802" alt="image" src="https://user-images.githubusercontent.com/36432/208747951-c6e27eaf-c56b-4f86-8018-19acac323fb5.png">

## Testing Instructions

1. Navigate to the Activity Log for an Atomic site that's been around for a little bit.
2. Verify the Server Actor appears as expected.
